src/ngraph/frontend/onnx_import/utils/broadcasting.cpp

```diff
@@ -38,22 +38,52 @@ static std::vector<ngraph::Shape> get_numpy_broadcast_shape(ngraph::Shape left_s
     auto rank_right = right_shape.size();
     auto max_rank = std::max(rank_left, rank_right);
 
-    for (auto i = 0; i < (max_rank - rank_left); ++i)
-    {
-        left_shape.insert(std::begin(left_shape), 1);
-    }
-    for (auto i = 0; i < (max_rank - rank_right); ++i)
-    {
-        right_shape.insert(std::begin(right_shape), 1);
-    }
-    for (auto index = 0; index < max_rank; ++index)
+    // left-pad the left_shape with ones
+    left_shape.insert(left_shape.begin(), max_rank - rank_left, 1);
+    // left-pad the right_shape with ones
+    right_shape.insert(right_shape.begin(), max_rank - rank_right, 1);
+
+    for (size_t index = 0; index < max_rank; ++index)
```

Comment:
Can you unify how we use `size_t` type? I mean in this file it is sometimes with namespace prefix `std::` sometimes without. Personally I prefer explicitly say it is from `std::`. Pleas add also `<cstddef>` header for this symbol.